### DISCUSSION TITLE
Highlighting for Angular html components

### DISCUSSION THREAD

**Discussion by m3astwood** [HiPhish/rainbow-delimiters.nvim#129] [Q&A]
Hello, 

I have angular components saved as `name.component.html` which are effectively plain html with some handlebar things mixed in. In neovim my angular components are detected as `angular.html` files - can I set rainbow to use the html parser for these files? 

I've read through the docs and tried a couple of things, but nothing is working out for me.

Thenks in advance!

**Comment by HiPhish** [comment]
I don't know Angular. There is an `angular` parser for Tree-sitter, so if someone can write the patterns and provide a sample file for tests it will be included. If you want to give it a try read the following topics:

- `:h rb-delimiters-custom-lang`
- `:h rb-delimiters-custom-query`

You will need to know how to write Tree-sitter queries. You can add the query to your own configuration first before opening a PR. That way you can use it immediately without having to wait for me to merge a PR.
